packages/hoppscotch-common/src/components/collections/MyCollections.vue

```diff
@@ -251,19 +251,36 @@
               <icon-lucide-search class="pb-2 opacity-75 svg-icons" />
             </template>
           </HoppSmartPlaceholder>
-          <HoppSmartPlaceholder
-            v-else-if="node === null"
-            :src="`/images/states/${colorMode.value}/pack.svg`"
-            :alt="`${t('empty.collections')}`"
-            :text="t('empty.collections')"
-          >
-            <HoppButtonSecondary
-              :label="t('add.new')"
-              filled
-              outline
-              @click="emit('display-modal-add')"
-            />
-          </HoppSmartPlaceholder>
+          <div v-else-if="node === null" class="flex flex-col space-y-25 py-5">
+            <div class="flex flex-col items-center space-y-4">
+              <span class="text-secondaryLight">
+                {{ t("collection.import_or_create") }}
+              </span>
+              <div class="flex space-x-4">
```

Comment:
When the width of the app is slightly smaller, the buttons begin to overflow sidewise. The second button should come underneath the first button in such cases

<img width="921" alt="image" src="https://github.com/hoppscotch/hoppscotch/assets/70131076/c88008c2-29e0-4064-8a93-276be9e78ee0">